### ISSUE TITLE
False positives with LightAntiCheat

### ISSUE THREAD

**Issue by tiredvekster** [Athlaeos/ValhallaMMO#14]
Hi! I'm LightAntiCheat's developer. I recently learned about false positives caused by skills of ValhallaMMO.
The issue: https://github.com/tiredvekster/LightAntiCheat/issues/6

My plugin has [an API](https://github.com/tiredvekster/LightAntiCheat/blob/main/F-A-Q.md#does-this-plugin-has-a-developer-api) to temporally disable the checks.
However, I encountered difficulties in figuring out how to detect the use of ValhallaMMO's skills.

Can you help me or create a hook for anti-cheats?

**Comment by Athlaeos** [comment]
Of course, could you name a couple of specific examples of what these false positives mean? 
For example, I would imagine flight gets flagged a lot when players happen to be using [double jumps](https://github.com/Athlaeos/ValhallaMMO/blob/master/core/src/main/java/me/athlaeos/valhallammo/listeners/JumpListener.java), which temporarily enable the player flight so that they can use the toggle flight control to jump in mid-air instead. 

KillAura might be triggered a lot too because critical hits with heavy weapons are by default configured to create [splash damage](https://github.com/Athlaeos/ValhallaMMO/blob/f645a8aa4dcef919f10790212fa7e4db693724a3/core/src/main/java/me/athlaeos/valhallammo/listeners/EntityAttackListener.java#L250) similar to sweeping edge

I think for most checks you can look at the [AccumulativeStatManager](https://github.com/Athlaeos/ValhallaMMO/blob/master/core/src/main/java/me/athlaeos/valhallammo/playerstats/AccumulativeStatManager.java) and see if the respective stat manipulating the vanilla game is not-zero, but there's probably a couple other things that will not be detectable using this stat manager. 

I reckon this would go a lot easier over discord, you can contact me at @athlaeos
